Gemfile.lock

```diff
@@ -754,4 +754,4 @@ RUBY VERSION
    ruby 2.1.5p273
 
 BUNDLED WITH
-   1.14.6
+   1.14.3
```

Comment:
If you squash that into the commit that changed this line in the first place, then you remove this little detour. But I actually don't mind the bundler version here when other parts of the file change as well.